concourse/tasks/forget-access-keys.yml

```diff
@@ -26,4 +26,8 @@ run:
         aws_iam_access_key.metrics_exporter
       echo "metrics exporter access key no longer managed by terraform."
 
+      terraform state rm -state=updated-tfstate/cf.tfstate \
+        aws_iam_access_key.yace
+      echo "yace access key no longer managed by terraform."
+
```

Comment:
We also need to add a corresponding section to the `remove-unused-ses-smtp-access-keys` task in the pipeline. This is also currently badly named because it currently handles both the ses-smtp user and the metrics user.

Reply:
I renamed the task to `remove-unused-iam-access-keys` and added the extra command for yace.